sphinx_gallery/scrapers.py

```diff
@@ -109,10 +109,13 @@ def matplotlib_scraper(block, block_vars, gallery_conf, **kwargs):
         Contains the configuration of Sphinx-Gallery
     **kwargs : dict
         Additional keyword arguments to pass to
-        :meth:`~matplotlib.figure.Figure.savefig`, e.g. ``format='svg'``.
-        The ``format`` kwarg in particular is used to set the file extension
-        of the output file (currently only 'png', 'jpg', 'svg', 'gif', and
-        'webp' are supported).
+        :meth:`~matplotlib.figure.Figure.savefig`, e.g. ``format='svg'``. The
+        ``format`` keyword argument in particular is used to set the file
+        extension of the output file (currently only 'png', 'jpg', 'svg',
+        'gif', and 'webp' are supported).
+
+        This is not used internally, but intended to be used when overriding
```

Comment:
```suggestion
        This is not used internally, but intended for used when overriding
```

Small nitpick

Reply:
That is not grammatically correct, unless you mean "for use when..."?

Reply:
Yes, sorry I've amended. I just want to make clearer that it is API available 'for' the user to use.